src/components/CommonMessage/CommonMessage.jsx

```diff
@@ -75,18 +70,19 @@ class CommonMessage extends PureComponent {
   }
 
   componentDidUpdate(prevProps) {
-    let propsMerged = this.props.headerMessages.concat(this.props.sideBarMessages);
-    let prevPropsMerged = prevProps.headerMessages.concat(prevProps.sideBarMessages);
-
     // Use lodash for a deep comparison of the merged messages.
-    if (!_.isEqual(propsMerged, prevPropsMerged)) {
-      this.checkToAssignTimer(propsMerged);
+    if (!_.isEqual(this.props.headerMessages, prevProps.headerMessages)) {
```

Comment:
Common Message no longer displays under the header. Looks like something broke in this commit.